README-source.md

```diff
@@ -269,18 +269,19 @@ zapier deprecate 1.0.0 2017-01-01
 
 ## Converting an Existing App
 
-> Warning! This is in a **very** alpha state - the immediate goal is to provide some basic structure to match an existing application. It will not even get close to a working copy of your existing app (yet).
-
-If you have an existing application (nominally named "V2") on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
+If you have an existing Web Build app on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
```

Comment:
`Web Build` -> `Web Builder`. 

Also I would probably hide the developer/builder link in markdown. Maybe something like `If you have an existing web builder app on the [Zapier site](https://zapier.com/developer/builder/), you can ...`